Formula/g/gimme-aws-creds.rb

```diff
@@ -3,10 +3,9 @@ class GimmeAwsCreds < Formula
 
   desc "CLI to retrieve AWS credentials from Okta"
   homepage "https://github.com/Nike-Inc/gimme-aws-creds"
-  url "https://files.pythonhosted.org/packages/fe/01/cdf8901f4c2a3f17974811e1e6669e655f7401111fa033bf07e5192086bd/gimme%20aws%20creds-2.7.0.tar.gz"
-  sha256 "a290e0de76c231a923c59fd2d7422331a94f64920aa972971d2c8ece7c461a23"
+  url "https://github.com/Nike-Inc/gimme-aws-creds/archive/refs/tags/v2.7.1.tar.gz"
```

Comment:
can you add code comment to reference [this open issue](https://github.com/Nike-Inc/gimme-aws-creds/issues/423)?